
# HG changeset patch
# User henryjen
# Date 1390363634 28800
# Node ID e8d4d0db1f06c26defcbf68e6713afab632f3f30
# Parent  c4b78cbea876aa86112cfda16c0026796848d582
8028407: adjust-mflags.sh failed build with GNU Make 4.0 with -I<path contains j>
Reviewed-by: erikj

diff -r c4b78cbea876 -r e8d4d0db1f06 make/bsd/makefiles/adjust-mflags.sh
--- a/make/bsd/makefiles/adjust-mflags.sh	Thu Dec 05 00:36:42 2013 -0800
+++ b/make/bsd/makefiles/adjust-mflags.sh	Tue Jan 21 20:07:14 2014 -0800
@@ -64,7 +64,7 @@
 	echo "$MFLAGS" \
 	| sed '
 		s/^-/ -/
-		s/ -\([^ 	][^ 	]*\)j/ -\1 -j/
+		s/ -\([^ 	I][^ 	I]*\)j/ -\1 -j/
 		s/ -j[0-9][0-9]*/ -j/
 		s/ -j\([^ 	]\)/ -j -\1/
 		s/ -j/ -j'${HOTSPOT_BUILD_JOBS:-${default_build_jobs}}'/
diff -r c4b78cbea876 -r e8d4d0db1f06 make/linux/makefiles/adjust-mflags.sh
--- a/make/linux/makefiles/adjust-mflags.sh	Thu Dec 05 00:36:42 2013 -0800
+++ b/make/linux/makefiles/adjust-mflags.sh	Tue Jan 21 20:07:14 2014 -0800
@@ -64,7 +64,7 @@
 	echo "$MFLAGS" \
 	| sed '
 		s/^-/ -/
-		s/ -\([^ 	][^ 	]*\)j/ -\1 -j/
+		s/ -\([^ 	I][^ 	I]*\)j/ -\1 -j/
 		s/ -j[0-9][0-9]*/ -j/
 		s/ -j\([^ 	]\)/ -j -\1/
 		s/ -j/ -j'${HOTSPOT_BUILD_JOBS:-${default_build_jobs}}'/
diff -r c4b78cbea876 -r e8d4d0db1f06 make/solaris/makefiles/adjust-mflags.sh
--- a/make/solaris/makefiles/adjust-mflags.sh	Thu Dec 05 00:36:42 2013 -0800
+++ b/make/solaris/makefiles/adjust-mflags.sh	Tue Jan 21 20:07:14 2014 -0800
@@ -64,7 +64,7 @@
 	echo "$MFLAGS" \
 	| sed '
 		s/^-/ -/
-		s/ -\([^ 	][^ 	]*\)j/ -\1 -j/
+		s/ -\([^ 	I][^ 	I]*\)j/ -\1 -j/
 		s/ -j[0-9][0-9]*/ -j/
 		s/ -j\([^ 	]\)/ -j -\1/
 		s/ -j/ -j'${HOTSPOT_BUILD_JOBS:-${default_build_jobs}}'/

